### ISSUE TITLE
IDE for project

### ISSUE THREAD

**Issue by huongtran** [benas/gamehub.io#15]
Hello Benas

Thank you for your great work

I am using your game hub, that is perfect, but i don't see any ide for project

Could you please give what ide you using to development this project ? is that eclipse ?

Thank you so much

Tran

**Comment by benas** [comment]
Hi @huongtran 

Thank you! I'm using notepad++.. No just kidding :smile: 
I use IntelliJ idea

Kind regards
Mahmoud

**Comment by huongtran** [comment]
Hi Benas

Thank you so much for you reply, 

nodeptad ++ LOL =))

Yes, i am using eclipse but it seem don't suggest, recommend function in Node, Express

IntelliJ  will display suggest function of Node, Express, ElasticSearch ?

**Comment by huongtran** [comment]
Yes, thank you so much for your reply, long time to see you reply

i am try up application to openshift to test some feature i development but openshift get more errors

that hard to config with openshift, could you please help me push code in openshift to new branch of github help me,

if you can help me, that is great, please help me this problem

looking forward to news from you

thank you so much
